src/azure-cli/HISTORY.rst

```diff
@@ -18,6 +18,7 @@ Release History
 * network nic ip-config add: Fixes #9861 where --ids was inadvertently exposed but did not work.
 * network application-gateway http-settings create/update: Fixes #9604. Add `--root-certs` to support user associate trusted root certificates with the HTTP settings.*
 * network dns record-set ns create: Fixes #9965. Support --subscription again by moving the supression into lower scope.*
+* network watcher test-ip-flow: Fixes #9845. Fixes #9844. Update doc and error messages.*
```

Comment:
This should be specific about what is actually fixed. No one is going to go back into Github history to find out what this means. Updating docs and error messages doesn't require a history entry.

Reply:
How about this one?